bedrock/externalpages/templates/externalpages/pocket/platform-pages.html

```diff
@@ -0,0 +1,31 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block page_title %} Add Pocket to Chrome {% endblock %}
+
+{% block pocket_css %}
+ {{ css_bundle('pocket-platforms') }}
+{% endblock %}
+
+{% block content %}
+  <div class="platform">
+    <div class="platform__content">
```

Comment:
Wondering if we're not using the `.platform__content` class if we need an extra container here?

Reply:
good eye!